src/main/java/com/thealgorithms/scheduling/entities/ProcessDetails.java

```diff
@@ -0,0 +1,56 @@
+package com.thealgorithms.scheduling.entities;
+
+public class ProcessDetails {
```

Comment:
If you think this class could be reused for other scheduling algorithms place it under the package [devutils](https://github.com/TheAlgorithms/Java/tree/master/src/main/java/com/thealgorithms/devutils), otherwise make it a private class and define it in the FCFSScheduling.java file.

Reply:
Have changed the location to devutils.